test/ovaldebug/main.go

```diff
@@ -1,95 +0,0 @@
-// Ovaldebug is a helper for debugging the ovalutil package.
```

Comment:
I was tempted to just remove the RHEL part of this, but without that, it essentially does nothing